sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaReadSchemaTransformProvider.java

```diff
@@ -41,19 +41,32 @@
 import org.apache.beam.vendor.guava.v26_0_jre.com.google.common.collect.Lists;
 import org.apache.kafka.clients.consumer.ConsumerConfig;
 import org.apache.kafka.common.serialization.ByteArrayDeserializer;
+import org.joda.time.Duration;
 
 @AutoService(SchemaTransformProvider.class)
 public class KafkaReadSchemaTransformProvider
     extends TypedSchemaTransformProvider<KafkaReadSchemaTransformConfiguration> {
 
+  final Boolean isTest;
+  final Integer testTimeoutSecs;
+
+  public KafkaReadSchemaTransformProvider() {
+    this(false, 0);
+  }
+
+  KafkaReadSchemaTransformProvider(Boolean isTest, Integer testTimeoutSecs) {
```

Comment:
Consider add a `@VisibleForTesting` annotation as this constructor is supposed to be used in test only?

Reply:
great point. done